tests/baselines/reference/docker/vscode.log

```diff
@@ -1,11 +1,148 @@
-Exit Code: 0
+Exit Code: 1
 Standard output:
 yarn run vX.X.X
 $ gulp compile --max_old_space_size=4095
 [XX:XX:XX] Node flags detected: --max_old_space_size=4095
 [XX:XX:XX] Using gulpfile /vscode/gulpfile.js
-Done in ?s.
+[XX:XX:XX] Error: /vscode/src/vs/workbench/contrib/codeEditor/browser/suggestEnabledInput/suggestEnabledInput.ts(229,3): Type 'string | null' is not assignable to type 'string'.
```

Comment:
@mjbvz this is a DOM change that will happen in 3.8. Can you pass this on the correct people? https://github.com/microsoft/TypeScript/pull/35207/files gives diff -- a number of properties now have type `string` instead of `string | null`.